.eslintignore

```diff
@@ -3,3 +3,4 @@
 __mocks__
 tests/__snapshots__
 coverage
+server-preload.js
```

Comment:
Note: I'm telling eslint to [ignore](https://eslint.org/docs/user-guide/configuring/ignoring-code) this file. Otherwise, it throws the following error:

```
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: server-preload.js.
The file must be included in at least one of the projects provided
```